root/edit/details/AddLabel.js

```diff
@@ -0,0 +1,125 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import isDateEmpty from '../../static/scripts/common/utility/isDateEmpty';
+import formatDate from '../../static/scripts/common/utility/formatDate';
+import yesNo from '../../static/scripts/common/utility/yesNo';
+import DescriptiveLink from
+  '../../static/scripts/common/components/DescriptiveLink';
+import formatIsni from '../../utility/formatIsni';
+
+type AddLabelProps = {
+  ...EditT,
+  +display_data: {
+    ...CommentRoleT,
```

Comment:
Technically the comment can be null here, but you can either make the suggested change or add `// ''` to `build_display_data`.
```suggestion
    +comment: string | null,
```